docs/config-definition.md

```diff
@@ -38,7 +41,17 @@ scheduleSyncPeriod: 1m
 restoreOnlyMode: false
 ```
 
-## Parameter Reference
+## Install in any namespace
+
+To install Ark in another namespace, you can specify the namespace in your `Config` YAML. That is, you can replace the value `heptio-ark` in the example with the appropriate namespace value.
```

Comment:
It's not just the Config yaml. It's all of the required resources that are created by the example yaml files:
- service account
- cluster-admin cluster role binding for service account
- ark role
- role binding for service account
- ark config
- ark deployment

Reply:
Yeah, this is all also not unrelated to issues #274 and #275, which will take more time. (but which I should be able to get into the release ... just not today). Right now there's no obvious place for any of this stuff, bc the example YAML files are just that, YAML only. Discuss in today's squad sync?